src/components/Modals/FiatRamps/FiatRampsCommon.ts

```diff
@@ -1,25 +1,4 @@
-import type { BigNumber } from 'lib/bignumber/bignumber'
-
 export enum FiatRampAction {
   Buy = 'buy',
   Sell = 'sell',
 }
-
-export type FiatRampAsset = {
```

Comment:
i can't even

Reply:
we didn't need anything other than an asset id, this was all random prop drilling/ugly icons/inconsistent names

Reply:
😔